src/webgpu/shader/validation/parse/switch.spec.ts

```diff
@@ -0,0 +1,75 @@
+export const description = `Validation parser tests for 'switch' statements`;
+
+import { makeTestGroup } from '../../../../common/framework/test_group.js';
+import { keysOf } from '../../../../common/util/data_tables.js';
+import { ShaderValidationTest } from '../shader_validation_test.js';
+
+export const g = makeTestGroup(ShaderValidationTest);
+
+const kTests = {
+  L_default: { wgsl: `switch L { default {} }`, pass: true },
+  L_paren_default: { wgsl: `switch (L) { default {} }`, pass: true },
+  L_case_1_default: { wgsl: `switch L { case 1 {} default {} }`, pass: true },
+  L_case_1_case_2_default: {
```

Comment:
please add a test case  with case1 then default then case2

Reply:
Done

Reply:
I don't see it?

Something like:
`switch 1 { case 1 {}  default {}  case 2 {} }`

Reply:
Sorry, I somehow interpreted your request as `case 2 {} case1{} default {}`. Done